services/auth-service/src/main/java/com/sourcefuse/jarc/services/authservice/controller/MyController.java

```diff
@@ -0,0 +1,18 @@
+package com.sourcefuse.jarc.services.authservice.controller;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class MyController {
```

Comment:
why this ?

Reply:
That was for testing auth protected routes will remove it